bin/linja.in

```diff
@@ -463,7 +463,7 @@ def make_build_ninja(args):
 
         print("""rule LEAN""", file=f)
         print("""  depfile = ${DLEAN_FILE}""", file=f)
-        print("""  command = "%s" %s $in -o "${OLEAN_FILE}" """ \
+        print("""  command = "%s" %s -o ${OLEAN_FILE} $in""" \
```

Comment:
The olean path should only include a space if the input file `$in` does.  In that case that line was already broken since `$in` was not quoted.